src/BootloaderVersion.cpp

```diff
@@ -1,26 +1,36 @@
 #include <cstdint>
+#include <cstdio>
 #include "BootloaderVersion.h"
 
 using namespace Pinetime;
 
-// NOTE : current bootloader does not export its version to the application firmware.
+// NOTE : version < 1.0.0 of bootloader does not export its version to the application firmware.
+
+uint32_t BootloaderVersion::version = 0;
+char BootloaderVersion::versionString[BootloaderVersion::VERSION_STR_LEN] = "0.0.0";
 
 uint32_t BootloaderVersion::Major() {
-  return 0;
+  return (BootloaderVersion::version >> 16u) & 0xff;
 }
 
 uint32_t BootloaderVersion::Minor() {
-  return 0;
+  return (BootloaderVersion::version >> 8u) & 0xff;
 }
 
 uint32_t BootloaderVersion::Patch() {
-  return 0;
+  return BootloaderVersion::version & 0xff;
 }
 
-const char* BootloaderVersion::VersionString() {
```

Comment:
I think this method can still return `const char*` as the caller is not supposed to modify the string. In fact, other getters should also probably be `const`.

Reply:
Thanks a lot for the feedback. I wrongly thought that a `const` method in C++ meant that the returned value was constant. I'll push a new version fixing that.